### ISSUE TITLE
extra motivation to ideas 

### ISSUE THREAD

**Issue by noxonsu** [noxonsu/dash.onout.org#119]
Noxon, [02.06.2022 16:31]
[In reply to Vitaliy Shulik]
думаю надо написать. we will charge 5$ for your vote and you will get 30% discount if product will be released.

Not Eternal, [02.06.2022 16:33]
[In reply to Noxon]
что то большая скидка

Noxon, [02.06.2022 16:33]
можно напистаь you will get 100$ discount

Noxon, [02.06.2022 16:33]
так уже понятно типа платит 5 баксов за 100

Noxon, [02.06.2022 16:33]
яб точно нажал))

Noxon, [02.06.2022 16:34]
еще снизу дописать "Do you have an idea post it in discord channel and if we release it you will get 200$ extra discount"

**Comment by VitaliyShulik** [comment]
@noxonsu а как проверять будем что купил адрес? Создать новый адрес для проверки голосований в дальнейшем и применяя скидку?

**Comment by noxonsu** [comment]
да уж как нибудь вручную в екслорере

**Comment by VitaliyShulik** [comment]
> да уж как нибудь вручную в екслорере

Да, и что-бы легче было проверять лучше создать отдельный адрес для этого))

**Comment by VitaliyShulik** [comment]
> да уж как нибудь вручную в екслорере

поэтому и говорю чтобы легче сделать проверку - использовать один адрес для сбора инфы по голосам и потом просто проверять скидку на определенные продукты, но вопрос опять в том как проверить за какой именно продукт был голос...

**Comment by NotEternal** [comment]
> оэтому и говорю чтобы легче сделать проверку - использовать один адрес для сбора инфы

можно сделать еще контракт, который хранит `account => products + voutes`. Будет проще, так-как не надо бонусы отсылать и делать оплату.

**Comment by VitaliyShulik** [comment]
> можно сделать еще контракт, который хранит `account => products + voutes`. Будет проще, так-как не надо бонусы отсылать и делать оплату.

Согласен, + будет бд тех кто купил продукты...

**Comment by NotEternal** [comment]
> > можно сделать еще контракт, который хранит `account => products + voutes`. Будет проще, так-как не надо бонусы отсылать и делать оплату.
> 
> Согласен, + будет бд тех кто купил продукты...

Проблема с оплатой еще не решена: надо делать все в одной транзакции, и оплату, и сохранение данных. Но думаю если делаем контракт для этой задачи, то можно подумать об оплате и добавить сюда часть функционала. Затем использовать его в упрощенном контракте оплаты.

> опять будем данные с нескольких сетей собирать)
